backend/views/syson-diagram-interconnection-view/src/main/java/org/eclipse/syson/diagram/interconnection/view/nodes/ChildPartUsageNodeDescriptionProvider.java

```diff
@@ -220,6 +223,49 @@ private NodeTool createNodeTool(NodeDescription nodeDescription, EClass eClass,
                 .build();
     }
 
+    private NodeTool createNodeToolWithDriection(NodeDescription nodeDescription, EClass eClass, NodeContainmentKind nodeKind, FeatureDirectionKind direction) {
```

Comment:
Is this method the same than the others in this PR?
If yes, could you share the code in a service class please?